app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(text: comment_params[:text], author_id: current_user.id, post_id: @post.id)
+
+    respond_to do |format|
+      format.html do
+        redirect_to user_post_path(@post.author.id, @post.id) if @comment.save
+      end
+    end
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```

Comment:
Remember to keep params private for securitty reasons [reference](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)

- [ ] Kindly make comment_params private